SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -220,16 +220,11 @@ void USpatialReceiver::HandlePlayerLifecycleAuthority(Worker_AuthorityChangeOp&
 
 void USpatialReceiver::HandleActorAuthority(Worker_AuthorityChangeOp& Op)
 {
-	if (Op.component_id == SpatialConstants::DEPLOYMENT_MAP_COMPONENT_ID)
-	{
-		GlobalStateManager->AuthorityChanged(Op.authority == WORKER_AUTHORITY_AUTHORITATIVE, Op.entity_id);
-		return;
-	}
+	StaticComponentView->OnAuthorityChange(Op);
```

Comment:
I've cleaned up the handling here a bit.  The SpatialReceiver now knows less about the GSM, reducing coupling.  Also, authority is handled independently for all components on the GSM, instead of just DEPLOYMENT_MAP

Reply:
Oh that's dope.